src/core/symbols/line_symbol.h

```diff
@@ -109,6 +109,17 @@ friend class OCAD8FileImport;
 		RoundJoin = 2
 	};
 	
+	/**
+	 * Mid symbol placement on dashed lines.
+	 */
+	enum MidSymbolPlacement
```

Comment:
Early comments... If `MidSymbolPlacement` were a scoped enum, the below identifiers could go without the leading `CenterOf`. It's natural that MidSymbol is in the centre.

Reply:
If "mid symbol" becomes "style symbol", we might need the `CenterOf.

Reply:
OK, no prob. The words `Dash` and `Gap` will anyway have to be dealt with as mid/style symbols apply on solid lines too.